meson_options.txt

```diff
@@ -23,3 +26,5 @@ option('use_sys_xxhash', type: 'boolean', value: false)
 option('use_sys_openssl', type: 'boolean', value: false)
 option('use_libuv', type: 'boolean', value: true)
 option('debugger', type: 'boolean', value: true)
+
+option('use_webui', type: 'boolean', value: false, description: 'install different WebUIs for radare2')
```

Comment:
False by default?

Reply:
me neither actually... also in fedora we don't ship it. For me it's ok to have it false by default. @radare ?

Reply:
we need to update the webui, but nobody seems to be in the mood to contribute, for me its ok to disable it because some distros get anoyed with licence issues because of being distribuetd in binary form